### PR TITLE
Bump HadoopFS version to 0.1.4 in docs

### LINKED ISSUE #2802: Docs point to stale HadoopFS and not the latest version
[Docs](https://docs.lakefs.io/integrations/spark.html#load-the-filesystem-jars) currently point to 0.1.3 as the latest version, although 0.1.4 is the latest.

### PR DESCRIPTION
**Pull request by arielshaqed**
Fixes #2802.

### CODE REVIEW

File: pkg/api/auth_middleware.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/getkin/kin-openapi/openapi3"
 	"github.com/getkin/kin-openapi/routers"
 	"github.com/getkin/kin-openapi/routers/legacy"
-	"github.com/golang-jwt/jwt"
+	"github.com/golang-jwt/jwt/v4"
```
Comment (itaiad200) [inline pkg/api/auth_middleware.go:13]: Is this change related or committed by accident?

**Review by itaiad200 — CHANGES_REQUESTED** [review]
I think there are some unrelated changes to this PR

**Review by nopcoder — CHANGES_REQUESTED** [review]
just discard the go changes, all the rest looks good.

**Review by nopcoder — APPROVED** [review]

**Comment by arielshaqed** [comment]
Yikes!  Tonnes of bad pushes.  Sorry; PTAL...

**Review by itaiad200 — APPROVED** [review]

**Comment by arielshaqed** [comment]
Thanks!
